src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoQuerier.cs

```diff
@@ -58,9 +58,9 @@ protected IDataReader GetDataReader(object data)
                 reader = ((Data.Tools.DataSets.DataTable)data).CreateDataReader();
             }
            
-            else if (data is DataSet)
+            else if (data is Microsoft.Data.Tools.DataSets.DataSet)

```

Comment:
I'll remove private "DataSets" dependency in a separate PR after getting initial conversion done.